Inserted Polygon Overview into Polygon Basics (#822)
maticnetwork/matic-docs: docs/validate/polygon-basics/what-is-polygon.md

- **Public sidechains**: Polygon sidechains are public in nature (vs. individual DApp chains), permissionless and capable of supporting multiple protocols.

The Polygon system was consciously architected to support arbitrary state transitions on Polygon sidechains, which are EVM-enabled.

## Delegator and Validator Roles

You can participate on the Polygon network as a delegator or validator. See:

* [Who is a Validator](/docs/validate/polygon-basics/who-is-validator)
* [Who is a Delegator](/docs/validate/polygon-basics/who-is-delegator)

## Architecture

If your goal is to become a validator, it is essential that you understand the Polygon architecture.

See [Polygon Architecture](/docs/validate/validator/architecture) for more information.

### Components

To have a granular understanding of Polygon's architecture, check out the core components:

* [Heimdall](/docs/contribute/heimdall/overview)
* [Bor](/docs/contribute/bor/overview)
* [Contracts](/docs/contribute/contracts/stakingmanager)

#### Codebases

> To have a granular understanding of the core components, see the codebases:

Reviewer: ```suggestion
To have a granular understanding of the core components, see the following codebases:
```

* [Heimdall](https://github.com/maticnetwork/heimdall)
* [Bor](https://github.com/maticnetwork/bor)
* [Contracts](https://github.com/maticnetwork/contracts)

## How-tos

### Node setup

> There are basically two ways of running a Validator Node at Polygon, with Ansible or from Binaries. You can check how to it with the links below:

Reviewer: ```suggestion
There are basically two ways of running a Validator Node on Polygon, using Ansible or directly from the binaries. You can check how to do this with the links below:
```
